frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -625,38 +740,65 @@ export function MaterializeVolumeAnnotationModal({
       </p>
     );
   }
+  const jobImage =
+    jobNameToImagePath[jobName] != null ? (
+      <>
+        <div style={{ textAlign: "center" }}>
+          <img
+            src={`/assets/images/${jobNameToImagePath[jobName]}`}
+            alt={`${jobName} example`}
+            style={{ width: 400, height: "auto", borderRadius: 3 }}
+          />
+        </div>
+        <br />
+      </>
+    ) : null;
 
   return (
-    <StartingJobModal
-      handleClose={handleClose}
+    <Modal
+      onCancel={handleClose}
+      open
+      width={700}
+      footer={null}
       title="Start Materializing this Volume Annotation"
```

Comment:
Again, it has been like this before, but could you please rename the modal title to make more sense, e.g. "Volume Annotation Materialization".